proto/viam/app/data/v1/data.proto

```diff
@@ -258,7 +258,7 @@ message ExportTabularDataResponse {
   string robot_name = 8;
   string robot_id = 9;
   string part_name = 10;
-  map<string, google.protobuf.Any> method_parameters = 11;
+  google.protobuf.Struct method_parameters = 11;
```

Comment:
I'm good with this change and think it is the right move long term, but @dmhilly, does it need to go through any review process as this is technically an API change?

Reply:
Just sending an email updating the stakeholders (doc reviewers) is fine. And they can voice any concerns/objections via email